<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UVATools</class>
 <widget class="QMainWindow" name="UVATools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UVATool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/UVATool_main_window.png</normaloff>../icons/UVATool_main_window.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="GraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="DrawingsToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Drawings</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="NodeAction"/>
   <addaction name="ElementAction"/>
  </widget>
  <widget class="QToolBar" name="ToolsToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="DrawAction"/>
  </widget>
  <widget class="QDockWidget" name="ChangeValues">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="NodeParameters">
       <property name="title">
        <string>Node Parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Fx</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fx">
              <property name="maximumSize">
               <size>
                <width>122</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Fy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fy">
              <property name="maximumSize">
               <size>
                <width>122</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>M</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m">
              <property name="maximumSize">
               <size>
                <width>122</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>P</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="p">
              <property name="maximumSize">
               <size>
                <width>122</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="primeiroGenero">
            <property name="text">
             <string>Primeiro Genero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="segundoGenero">
            <property name="text">
             <string>Segundo Genero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="terceiroGenero">
            <property name="text">
             <string>Terceiro Genero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="semiRigido">
            <property name="text">
             <string>Semi RÃ­gio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="semApoio">
            <property name="text">
             <string>Sem Apoio</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="confirmButton">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="ElementAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/element.png</normaloff>../icons/element.png</iconset>
   </property>
   <property name="text">
    <string>element</string>
   </property>
   <property name="toolTip">
    <string>Element</string>
   </property>
  </action>
  <action name="NodeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/node.png</normaloff>../icons/node.png</iconset>
   </property>
   <property name="text">
    <string>node</string>
   </property>
   <property name="toolTip">
    <string>Node</string>
   </property>
  </action>
  <action name="DrawAction">
   <property name="icon">
    <iconset>
     <normaloff>../icons/draw.png</normaloff>../icons/draw.png</iconset>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
   <property name="toolTip">
    <string>Draw Menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
